crates/bevy_ui/src/ui_node.rs

```diff
@@ -260,38 +261,168 @@ impl Val {
+/// See below for general learning resources and for documentation on the individual style properties.
+///
+/// ### Flexbox
+///
+/// - [MDN: Basic Concepts of Grid Layout](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Basic_Concepts_of_Grid_Layout)
+/// - [A Complete Guide To Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) by CSS Tricks. This is detailed guide with illustrations and comphrehensive written explanation of the different Flexbox properties and how they work.
+/// - [Flexbox Froggy](https://flexboxfroggy.com/). An interactive tutorial/game that teaches the essential parts of Flebox in a fun engaging way.
+///
+/// ### CSS Grid
+///
+/// - [MDN: Basic Concepts of Flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox)
+/// - [A Complete Guide To CSS Grid](https://css-tricks.com/snippets/css/complete-guide-grid/) by CSS Tricks. This is detailed guide with illustrations and comphrehensive written explanation of the different CSS Grid properties and how they work.
+/// - [CSS Grid Garden](https://cssgridgarden.com/). An interactive tutorial/game that teaches the essential parts of CSS Grid in a fun engaging way.
+
 #[derive(Component, Clone, PartialEq, Debug, Reflect)]
 #[reflect(Component, Default, PartialEq)]
 pub struct Style {
-    /// Whether to arrange this node and its children with flexbox layout
+    /// Which layout algorithm to use when laying out this node's contents:
+    ///   - [`Display::Flex`]: Use the Flexbox layout algorithm
+    ///   - [`Display::Grid`]: Use the CSS Grid layout algorithm
+    ///   - [`Display::None`]: Hide this node and perform layout as if it does not exist.
     ///
-    /// If this is set to [`Display::None`], this node will be collapsed.
+    /// <https://developer.mozilla.org/en-US/docs/Web/CSS/display>
     pub display: Display,
-    /// Whether to arrange this node relative to other nodes, or positioned absolutely
+
+    /// Whether a node should be laid out in-flow with, or independently of it's siblings:
+    ///  - [`PositionType::Relative`]: Layout this node in-flow with other nodes using the usual (flexbox/grid) layout algorithm.
+    ///  - [`PositionType::Absolute`]: Layout this node on top and independently of other nodes.
+    ///
+    /// <https://developer.mozilla.org/en-US/docs/Web/CSS/position>
     pub position_type: PositionType,
+
+    /// Whether overflowing content should be displayed or clipped.
+    ///
+    /// <https://developer.mozilla.org/en-US/docs/Web/CSS/overflow>
+    pub overflow: Overflow,
+
+    /// Defines the text direction. For example English is written LTR (left-to-right) while Arabic is written RTL (right-to-left).
+    ///
+    /// Note: the corresponding CSS property also affects box layout order, but this isn't yet implemented in bevy.
+    /// <https://developer.mozilla.org/en-US/docs/Web/CSS/direction>
+    pub direction: Direction,
```

Comment:
Hmm... perhaps. CSS does use `direction` primarily to control the text-direction (https://developer.mozilla.org/en-US/docs/Web/CSS/direction), much like it uses `color` for text color. But flexbox/grid are also supposed to flip the box layout direction too and don't currently (because direction isn't implemented at all in Taffy atm).

In any case, that property isn't new in this PR, so I think it would probably make sense to change separately?